app/scripts/proactive/rest/studio-client.js

```diff
@@ -197,7 +197,11 @@ define(
                             }
                         },
                         error: function(response) {
-                            console.error("Unknown User Permission", response)
+                            console.error("Unknown User Permission", response);
+                            var connectionError = $('<div id="connection-error" class="btn-group pa-tools"><label style="color:red; font-weight:bold; font-size:15px; padding-left:4px; padding-top:4px;">The server cannot be reached. Check the server status and your network connection. </label></div>');
```

Comment:
I think it's not the right place, you should move it to the ligne 183

Reply:
if I put it to line 183, it won't be executed in the case when the server is not running